js/lib/ui.js

```diff
@@ -35,6 +35,18 @@ elgg.ui.init = function () {
 	if ($('.elgg-autofocus').length > 0) {
 		elgg.deprecated_notice('Use of .elgg-autofocus is deprecated by html5 autofocus', 1.9);
 	}
+    $(document).on('click', '.elgg-user-settings-edit', function(){
```

Comment:
looks like incorrect indents... at various locations... also the accolades should be preceded by a space... probably we do not have code quality checks on the JS, so that is why automated tests do not scream about this...